Formula/todoman.rb

```diff
@@ -75,6 +75,9 @@ class Todoman < Formula
 
   def install
     virtualenv_install_with_resources
+
+    bash_completion.install "contrib/completion/bash/_todo" => "todo"
+    zsh_completion.install "contrib/completion/zsh/_todo"
```

Comment:
ZSH completion requires jq in certain circumstances, and due to a bug, can currently fail if it is not installed (see https://github.com/pimutils/todoman/issues/498). Should I do something about this?

Also, note that there are no Fish completions at the moment.

Reply:
I'd suggest to add JQ as a build dependency then

Reply:
Sounds like it should be a runtime dep

Reply:
It would be an unnecessary dependency for people not using other shells, but that's fine I guess.